Traducir.Core/Services/SOStringService.cs

```diff
@@ -30,6 +30,7 @@ public interface ISOStringService
         Task PullSODump(string dumpUrl);
         Task UpdateTranslationsFromSODump();
         Task<bool> ManageUrgencyAsync(int stringId, bool isUrgent, int userId);
+        Task<bool> DeleteSuggestionAsync(SOStringSuggestion suggestion);
```

Comment:
following the other methods from here, have it receive a `suggestionId`. I would also receive a `userId` so that in the future, other callers don't just call it and we end up adding a history saying that it was `deletedByOwner`.

On the implementation, you can return `false` if the `userId` doesn't match the owner (and if in the future we want other processes/people deleting suggestions, they will have to deal with the `SuggestionHistory` changes)

Reply:
so, I only store on history if the user match the user from the suggestion? if that is not the case, I forbid the delete? or I allow the delete but not store on history?

Reply:
the delete process should fail... as right now, we don’t have a way to delete suggestions not by its owner.

Reply:
great...